__tests__/app.test.js

```diff
@@ -13,12 +13,15 @@ afterAll(() => {
     database.end()
 })
 
-// be available on /api/topics
-// get all topics
 
-// it should respond with an array
-// the array should contain topic objects
-// each object should have property of slug, description
+describe("404 Upon Bad Path", () => {
+    it("should respond with 404 upon bad path", () => {
+        return request(app)
+        .get('/invalidpath')
+        .expect(404)
+    })
```

Comment:
Currently this test is passing as you are using Express built in 404 status for an endpoint that doesn't exist. However, this won't provide a message to the user. It would be nice to see a message being send. Something like "Invalid endpoint"